service/config.go

```diff
@@ -336,8 +336,15 @@ func (c *NitroClient) DeleteResource(resourceType string, resourceName string) e
 //DeleteResourceWithArgs deletes a resource of supplied type and name. Args are supplied as an array of strings
 //Each array entry is formatted as "key:value"
 func (c *NitroClient) DeleteResourceWithArgs(resourceType string, resourceName string, args []string) error {
+	//Remove unwanted argument (username) for listing but keep it for delete operation
+	var err error
+	if resourceType == "snmptrap_snmpuser_binding" {
+		get_args := args[:len(args)-1]
```

Comment:
Please iterate over the `args` array and remove the item with the `username` key.